dissect/target/plugins/os/windows/lnk.py

```diff
@@ -168,7 +168,18 @@ def lnk(self, directory: Optional[str] = None) -> Iterator[LnkRecord]:
 
     def lnk_entries(self, path: Optional[str] = None) -> Iterator[TargetPath]:
         if path:
-            yield self.target.fs.path(path)
+            target_path = self.target.fs.path(path)
+            if not target_path.exists():
+                self.target.log.error("Provided path does not exist on target")
+                return
+            if target_path.is_file() and target_path.suffix.lower() != ".lnk":
+                self.target.log.error("Provided file should have suffix '.lnk'")
```

Comment:
The `Lnk` class already handles 'invalid' lnk files by raising an error.  I think its better to just try and parse the file than quitting out on a suffix check, especially when the user explicitly points to a file.

Reply:
All good, I removed this check